src/libc_installation.cpp

```diff
@@ -111,7 +111,7 @@ Error zig_libc_parse(ZigLibCInstallation *libc, Buf *libc_file, const ZigTarget
     }
 
     if (buf_len(&libc->msvc_lib_dir) == 0) {
-        if (target->os == OsWindows) {
+        if (target->os == OsWindows && target->abi == ZigLLVM_MSVC) {
```

Comment:
For MinGW/MSVC detection, can we do it with this logic?

```c++
bool is_mingw(ZigTarget *target) {
    return target->os == OsWindows and target_abi_is_gnu(target);
}
```

And then if it's not mingw, we assume MSVC.

Reply:
i think only the `target_abi_is_gnu(target->abi)` is necessary as that would fit better in to more places such as `zig_libc_parse` and `zig_libc_find_native`.